docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-237.adoc

```diff
@@ -0,0 +1,45 @@
+
+== Dedicated data endpoint is disabled
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/ACRDedicatedDataEndpointEnabled.py[CKV_AZURE_237]
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is checking to ensure that dedicated data endpoints are enabled in Azure. Dedicated data endpoints represent an enhanced level of security in a cloud environment by ensuring that data connections occur directly through private IP addresses within your virtual network. If dedicated data endpoints are not enabled, it could pose a security risk by allowing data to traverse over the public internet, increasing the risk of data interception or breaches.
+
+=== Fix - Buildtime
+
+*Terraform*
+
+* *Resource:* azurerm_container_registry
+* *Arguments:* data_endpoint_enabled
+
+To fix this issue, you need to enable the dedicated data endpoints. This option enhances the security of your solution by limiting data traffic to private IP addresses.
```

Comment:
Enable the dedicated data endpoints to limit data traffic to private IP addresses in your virtual network.